packages/web-client/app/components/card-pay/dashboard-panel/section/index.css

```diff
@@ -59,6 +59,70 @@
   color: var(--boxel-purple-500);
 }
 
+.dashboard-panel-section__badges {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  margin-top: var(--boxel-sp-lg);
+  margin-bottom: var(--boxel-sp-xl);
+}
+
+.dashboard-panel-section__badges > div {
+  flex-basis: 50%;
+  height: 3.6rem;
+}
+
+.dashboard-panel-section__badges > div:last-child {
+  text-align: center;
+}
+
+.dashboard-panel-section__badges img {
+  height: 2.8rem;
+  object-fit: contain;
+}
+
+.dashboard-panel-section__badges-google-play-badge {
+  opacity: 0.3;
+}
+
+.dashboard-panel-section__badges-google-play-soon {
+  font-size: var(--boxel-font-size-xs);
+  text-align: center;
+}
+
+.dashboard-panel-section__purchase-card-phone {
+  display: block;
+}
+
+.dashboard-panel-section-wallet {
+  justify-content: flex-start;
+}
+
+.dashboard-panel-section-wallet__cards {
+  width: calc(100% + (2 * 84px));
+  position: absolute;
+  bottom: 0;
+  left: -84px;
+}
```

Comment:
We should probably move all this wallet CSS to the wallet.css file and rename the classes accordingly.

Reply:
Good idea.